www/js/controller.js

```diff
@@ -575,6 +575,12 @@ module.exports = {
     global.platform.ipc.send('show-empty-slide');
     global.platform.ipc.send('show-text', { text, isGurmukhi, isAnnouncement });
   },
+  sendMimicShabad(mimicShabad) {
```

Comment:
please call the function `sendTextWithTranslations` instead of "sendMimicShabad"